### DISCUSSION TITLE
draw + type = "simultaneous"

### DISCUSSION THREAD

**Discussion by AlainZuur** [gavinsimpson/gratia#313] [General]
Hello...what would be the easiest way to get simultaneous CIs with draw()? I can get them manually with confint...but it would be much easier if draw() could plot them (I don't think it can??).

I can see how to do it manually:
https://gavinsimpson.github.io/gratia/articles/custom-plotting.html
This part:
sm <- smooth_estimates(m) |> add_confint()
And then replace by: confint  and adjust the type argument. Is there any way of doing this faster?

Or change that line in draw.gam?
Alain

**Comment by gavinsimpson** [comment]
Hi Alain,

As of now, there isn't a simpler way than `smooth_estimates() |> add_confint()` and replacing the confidence interval with `confint()` (or fixing `draw.gam` in the `"gratia"` namespace to add the functionality you want).

I hadn't thought much about this for the `draw.gam()` method as the "across-the-function" nature of the frequentist coverage of the standard Bayesian credible intervals is likely good enough for most intended uses of this function. But I had noticed that `add_confint()` lacked simultaneous intervals and have thought how to add this (it would need the model to be passed in and some extra work). IF I implemented that, then `draw.gam()` could gain the functionality you want.

**Reply by AlainZuur** [reply]
Hello Gavin, 

Thanks for the feedback. Appreciated! Changing the crit value (manually) in one of those 'add' functions does the job.  

Great package, by the way.

Kind regards,

Alain
